lib/modules/versioning/debian/readme.md

```diff
@@ -0,0 +1,4 @@
+Debian versioning is used for Debian container images that are referenced by its major release version or a codename.
+
+Versions to which this scheme applies are e.g. `11` and `bullseye`.
```

Comment:
i think 11.4 should work too

Reply:
The current impl doesn't cover it. I've now added `11.4` as not-compliant too, as this is indeed not obvious or intuitive.